scrapetube/scrapetube.py

```diff
@@ -182,9 +182,8 @@ def get_videos(
 
 def get_initial_data(session: requests.Session, url: str) -> str:
     response = session.get(url)
-    if "uxe=" in response.request.url:
-        session.cookies.set("CONSENT", "YES+cb", domain=".youtube.com")
-        response = session.get(url)
+    session.cookies.set("CONSENT", "YES+cb", domain=".youtube.com")
+    response = session.get(url)
```

Comment:
this will download everything twice, can you remove the first `session.cookies.set` ?

Reply:
Oh yeah, my bad, just removed the line.

Reply:
no problem man, good luck!!